spec/fields/name_spec.rb

```diff
@@ -21,6 +21,7 @@ def mods_display_name(mods_record)
     @numeral_toa = Stanford::Mods::Record.new.from_str(numural_toa_fixture).plain_name
     @simple_toa = Stanford::Mods::Record.new.from_str(simple_toa_fixture).plain_name
     @display_form = Stanford::Mods::Record.new.from_str(display_form_name_fixture).plain_name
+    @entities = Stanford::Mods::Record.new.from_str(entities_name_fixture).plain_name
```

Comment:
Can this move down into a describe block? Then we don't pollute the global space with this ivar that is only ever used the one time.